### ISSUE TITLE
How to finetune a model with some weights fronzen and some weights having different learning rate?

### ISSUE THREAD

**Issue by brappier** [liuliu/s4nnc#10]
Is there any example?

**Comment by liuliu** [comment]
See https://github.com/liuliu/swift-diffusion/blob/main/examples/lora_training/main.swift and https://github.com/liuliu/swift-diffusion/blob/main/src/LoRA.swift#L434

**Comment by brappier** [comment]
Thanks
